### PR TITLE
Sb 14372

### PR DESCRIPTION
**Pull request by Hari-stackroute**
Telemetry related changes to merge user

### CODE REVIEW

File: sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java [hunk 1]
```diff
@@ -919,7 +919,7 @@
   public static final String ES_OR_OPERATION = "$or";
   public static final String PREV_USED_EMAIL ="prevUsedEmail";
   public static final String PREV_USED_PHONE ="prevUsedPhone";
-  public static final String MERGE = "merge";
+  public static final String MERGE_USER = "Mergeuser";
```
Comment (manzarul) [inline sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java]: Mergeuser to MergeUser

File: sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java [hunk 2]
```diff
@@ -958,6 +957,8 @@
   public static final String TELEMETRY_EDATA_USER_MERGE_ACTION = "merge-user-courses-and-cert";
   public static final String BE_JOB_REQUEST = "BE_JOB_REQUEST";
   public static final String TELEMETRY_ACTOR_USER_MERGE_ID = "Merge User Courses and Cert";
-  public static final String SUNBIRD_ACCOUNT_MERGE_BODY  = "sunbird_account_merge_body";
+  public static final String CERTIFICATE = "Certificate";
+  public static final String MERGE_CERT = "Mergecert";
```
Comment (manzarul) [inline sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java]: Mergecert to MergeCert

Comment (Hari-stackroute) [reply sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java]: We are using this in telemetry, there only first character is captial and all remaining is lower-case

File: sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java (hunk 1 above)
Comment (Hari-stackroute) [reply sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java]: We are using this in telemetry, there only first character is captial and all remaining is lower-case

**Pull request merged**